src/ipc-handlers.ts

```diff
@@ -449,7 +449,12 @@ export async function getAppGlobals( _event: IpcMainInvokeEvent ): Promise< AppG
 	};
 }
 
-export async function getWpVersion( _event: IpcMainInvokeEvent, wordPressPath: string ) {
+export async function getWpVersion( _event: IpcMainInvokeEvent, id: string ) {
```

Comment:
Should we update https://github.com/Automattic/studio/blob/28aa115e35d138fe04f51b37e13ff0eb30042d2d/src/preload.ts#L32 as well?

Reply:
Yes!, thanks for the early feedback.

Changed https://github.com/Automattic/studio/pull/36/commits/691fa9a421a4f2babae9582d4d686c63ecfe15df